automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -449,4 +435,21 @@ final class LeonardoAzureSuite
       new AzureDiskSpec
     )
     with TestSuite
+    with AzureBilling
     with ParallelTestExecution
+    with BeforeAndAfterAll {
+  override def beforeAll(): Unit = {
```

Comment:
oh so this had to be within the LeonardoAzureSuite class to work correctly?

Reply:
I don't think it has to be here, it just cannot be in the same trait as the fixture or it will get called by everything that mixes in the fixture